src/main/java/s25/cs151/application/DefineSemesterHoursController.java

```diff
@@ -22,7 +22,24 @@ public void initialize() {
         // Create the FormsFX form
         Form form = Form.of(
             Group.of(
-                    Field.ofSingleSelectionType(Arrays.asList("Spring", "Summer", "Fall", "Winter"), 1).label("Semester")
+                    Field.ofSingleSelectionType(Arrays.asList("Spring", "Summer", "Fall", "Winter"), 0).label("Semester"),
+                    Field.ofStringType("2025")
```

Comment:
should use an integer control here

```suggestion
                    Field.ofIntegerType(2025)
```